### PR TITLE
Give the card component an info property

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

[Give the card component an info property](https://github.com/webkom/lego-webapp/commit/3cc4dc46d7942043df2c6040bd21056f6b9ecb72)

---
  
[Utilize the info card](https://github.com/webkom/lego-webapp/commit/755ba4a654521799790a5883e1778581b352aae8)

# Result

image

image

---

Also resolves ABA-222

### CODE REVIEW

**Review by eikhr — APPROVED** [review]
Looks good:)

File: app/components/Card/index.tsx [hunk 1]
```diff
@@ -13,13 +13,42 @@ const CardHeader = ({ children, className }: CardHeaderProps) => (
...
+  let icon;
+  if (danger) {
+    icon = <Icon name="alert-circle-outline" className={styles.warningIcon} />;
+  }
+
+  if (info) {
+    icon = (
+      <Icon name="information-circle-outline" className={styles.infoIcon} />
+    );
+  }
```
Comment (itsisak) [inline app/components/Card/index.tsx:32]: I think maybe `if (info)` should come before `if (danger)` that way if both are true, it is going to be a danger card

**Review by itsisak — APPROVED** [review]
LGTM

File: app/components/Card/index.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/components/Card/index.tsx:32]: That's a valid point. I'm planning on adding `warning` as well, and when I do I think I'll merge these into one prop so you'll never actually get to pass multiple severities.

**Pull request merged**